gapis/resolve/dependencygraph2/graph_visualization/graph_structure.go

```diff
@@ -197,122 +189,6 @@ func (input nodeSorter) Less(i, j int) bool {
 	return input[i].id < input[j].id
 }
 
-func (g *graph) traverseGraph(currentNode *node, visitTime, minVisitTime, idInStronglyConnectedComponents, visitedNodesId *[]int, currentId, currentTime *int) {
```

Comment:
When moving big chunks of code like this, it's usually preferred to do the move in a separate commit, since it is hard to see what changed. Just a comment, no need to change this.

Reply:
I have abstracted this part of code to simplify the understanding, because it is essentially the Tarjan algorithm to compute Strongly Connected Components,